core/build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
 
     implementation "com.google.protobuf:protobuf-java:${revProtoBuf}"
 
+    implementation "javax.ws.rs:jsr311-api:${revJsr311Api}"
```

Comment:
now that HTTP is in its own module, is this dependency still needed?

Reply:
fixed